### ISSUE TITLE
I have great suggestion for pi apps

### ISSUE THREAD

**Issue by CoderLogy** [Botspot/pi-apps#692]
Make a website or something in your gut where u users can submit their apps they have made or searched on the net and when you find the app is worthing u put it on pi apps like what happens on the play store instead of attending to so many issues related to app suggestion u got app suggestion in write format and its easier for dev wanting apps to put on pi apps

**Comment by Itai-Nelken** [comment]
I'm not sure I understand. do you mean a option in pi-apps where you can directly submit app requests?

**Comment by CoderLogy** [comment]
yes I meant botspot should make a GUI or something like that for developers to put the app on that GUI or website and if botspot find app worthing to put it on pi-apps he would do like their play store verification system in which when devs put the app on play store the google tick or crosses app checking is it worth putting thus it would reduce the number of issues u get on pi apps related to app suggestion where people could directly suggest

**Comment by Botspot** [comment]
> yes I meant botspot should make a GUI or something like that for developers to put the app on that GUI or website and if botspot find app worthing to put it on pi-apps he would do like their play store verification system in which when devs put the app on play store the google tick or crosses app checking is it worth putting thus it would reduce the number of issues u get on pi apps related to app suggestion where people could directly suggest

I agree with you: there is a serious bottleneck  in the system of app-submission. But I don't think this bottleneck is where you think it is, and I don't think it's as easy to solve as you think.

In my opinion, app suggestions and submissions get slowed down for three main reasons:

1. The app submissions are **low quality**. It's very rare for an app to be perfect the first time it's submitted. Almost always, there are significant issues with it, from script problems to things like "Please write a better description for this". All of this feedback takes time for everyone involved, causing further delays for future apps.
2. The app submitters are **not motivated**. This ties in to number 1. A significant portion of people feel as though it's the Pi-Apps devs' job to fix the details, write the better description, improve the icon. Some people come to the issues page with the mindset that we're "eager" and "looking" for new apps, and will _gladly create the whole app_ for them.
3. The Pi-Apps devs **are busy**. With every app submission, there is a built-in time cost in dealing with it from start to finish. And after dealing with hundreds and hundreds of these requests, it is hard to stay motivated! In the best case scenario, an app will need to be tested. In the worst case, it requires seemingly endless testing, debugging and polishing before it's ready. This busyness factor is not helped one bit by my decision to work at a summer camp during this summer.

An interface-based app submission process would be great. But I can't see how it can work for Pi-Apps.
We have to remember that the average person who submits an app request is:

- A kid.
- Unpaid.
- Not that good at Linux, or doesn't care.
- Doesn't really gain anything in return for submitting the app.

Compare this to the "big" app stores like Apple's or Windows or Android's, where the average app-submitter is:

- An adult. (Or a whole company)
- Paid.
- Skilled with programming. (Or has a whole team of developers)
- Determined to push it through the system because their app will generate money.

Bottom line: Pi-Apps is just so different than any money-making app store, that I don't really see how their methods would work here.

**Comment by CoderLogy** [comment]
no botspot i meant to just ease your work there should be a website or kind of thing where people submit their apps and plus it would be easy for u to handle if u see long term plus u can add there reviews of what new apps people want thus making your and people job easier plus it looks good when there is no issue on your GitHub repo, so kind of think how good is this in long term

**Comment by Botspot** [comment]
> no botspot i meant to just ease your work there should be a website or kind of thing where people submit their apps and plus it would be easy for u to handle if u see long term plus u can add there reviews of what new apps people want thus making your and people job easier plus it looks good when there is no issue on your GitHub repo, so kind of think how good is this in long term

Github.com: mature, modern, and trusted website where millions already have accounts to submit apps.
A custom website: less mature, would probably look uglier, untrusted, and 100% of people who wanted to submit an app would have to create some form of unique user ID.

I think that while logical, creating our own app-creation website would save less time than it consumes. (And plus, websites usually cost money)
